sale_order_type/models/sale_order_type.py

```diff
@@ -0,0 +1,22 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+# For copyright and license notices, see __openerp__.py file in root directory
+##############################################################################
+
+from openerp import fields, models
+
+
+class SaleOrderTypology(models.Model):
+    _name = 'sale.order.type'
+    _description = 'Type of sale order'
+
+    name = fields.Char(string='Name', required=True)
+    description = fields.Text(string='Description')
+    sequence_id = fields.Many2one(
+        comodel_name='ir.sequence', string='Entry Sequence', copy=False)
+    journal_id = fields.Many2one(
+        comodel_name='account.journal', string='Billing Journal')
+    refund_journal_id = fields.Many2one(
+        comodel_name='account.journal', string='Refund Billing Journal')
```

Comment:
On _prepare_refund use "journal = pickings[0].sale_id.type_id.refund_journal_id" and then "journal.id", maybe better to set as required=True